sendcodes.py

```diff
@@ -24,6 +25,7 @@
 # Tableau includes a blank column. Which is cool.
 BLANK = 4
 
+timezone = pytz.timezone('America/Denver')
```

Comment:
It seems like this could be in the config file so that other locations could override this without modifying the code. I guess for backwards compatability it should default to `America/Denver` if no value is in the config.

Reply:
Good point.  I prefer to require that the value be explicitly set and fail fast if it's not present.  Will need to make sure that it is added to the prod configuration on the next deployment, though.

Reply:
If it's not defined, you get this error, which is fairly self-explaining:

```
Traceback (most recent call last):
  File "sendcodes.py", line 105, in <module>
    tzOffset = TIMEZONE.utcoffset(parsed_date).total_seconds() / 60
NameError: name 'TIMEZONE' is not defined
```

Reply:
Seems great to me, thanks!